src/index.js

```diff
@@ -9,5 +9,39 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+$( document ).ready(function() {
 
-console.log('This is the JavaScript entry file - your code begins here.');
+$('.login-button').on( "click", function() {
```

Comment:
This is a nit picky thing but to eliminate any possible characters, this could also be accomplished with the syntax '$('.login-button').click( callback )